rosdep/base.yaml

```diff
@@ -715,6 +715,7 @@ g++-static:
   arch: [gcc]
   debian: [g++]
   fedora: [gcc-c++, glibc-devel, glibc-static, libstdc++-devel, libstdc++-static]
+  gentoo: [sys-devel/gcc]
   ubuntu: [g++]
```

Comment:
This entry needs a total refactoring:

1. The name is wrong, there is no gcc-static in fedora, only glibc-static
2. remove all other distros other than fedora